app/collections/Classrooms.coffee

```diff
@@ -0,0 +1,17 @@
+Classroom = require 'models/Classroom'
+CocoCollection = require 'collections/CocoCollection'
+
+module.exports = class Classrooms extends CocoCollection
+  model: Classroom
+  url: '/db/classroom'
+  
+  initialize: ->
+    @on 'sync', ->
+      for classroom in @models
+        classroom.capitalizeLanguageName()
```

Comment:
I don't think this will work, because `this` is not set. Either switch to `@listenTo @, 'sync', ->` or `@on 'sync', =>`.